src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -90,12 +93,12 @@ const ApplicationLauncher = _.flow(
     return h(Modal, {
       onDismiss,
       width: 530,
-      title: _.size(outdatedAnalyses) > 1 ? 'R Markdown Files In Use' : `R Markdown File Is In Use`,
+      title: _.size(outdatedAnalyses) > 1 ? 'R files In Use' : `R file Is In Use`,
```

Comment:
Should these messages be sentence cased?

```suggestion
      title: _.size(outdatedAnalyses) > 1 ? 'R files in use' : `R file is in use`,
```